src/cookies/cookies.test.ts

```diff
@@ -1,5 +1,9 @@
 import MockDate from 'mockdate';
-import * as cookies from './index';
+import { getCookie } from './getCookie';
+import * as getCookieValues from './getCookieValues';
```

Comment:
Could we `import { getCookieValues }` here to make the import explicit? Or is this necessary to support jest's `spyOn` API?